kernel/examples/common/Cargo.toml

```diff
@@ -1,7 +1,13 @@
 [package]
 name = "common"
-version = "0.1.0"
-edition = "2021"
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+readme.workspace = true
+version.workspace = true
+
+[package.metadata.release]
+release = false
```

Comment:
Seems that the `package.metadata` table is not checked by cargo and is used for external tools. Could you add a comment that mentions what tool uses this?

Reply:
Got it. 👍  https://github.com/crate-ci/cargo-release/blob/master/docs/reference.md